analyzers/src/SonarAnalyzer.Common/Rules/MultipleVariableDeclarationBase.cs

```diff
@@ -18,50 +18,40 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Diagnostics;
-using SonarAnalyzer.Helpers;
-
 namespace SonarAnalyzer.Rules
 {
-    public abstract class MultipleVariableDeclarationBase : SonarDiagnosticAnalyzer
+    public struct MultipleVariableDeclarationConstants
```

Comment:
What's the benefit of `struct` over `static class`?

C# doesn't have `Module`, unfortunately 😄

Reply:
what is the problem with struct?

Reply:
It raises a maintainability question about why it needs to be something unusual as a struct for this purpose